ui/components/app/wallet-overview/eth-overview.js

```diff
@@ -118,6 +120,7 @@ const EthOverview = ({ className, showAddress }) => {
   }
 
   const isSwapsChain = useSelector(getIsSwapsChain);
+  const signingDisabled = !account.methods.includes(EthMethod.SignTransaction);
```

Comment:
We could add a new selector to follow the same "style" than other "enablers" here, like a `getIsSigningAllowed` maybe?

Also, to follow current file pattern/flow, we could use "enabled" instead of "disabled", WDYT?

Reply:
Good point! We seem to already have a selector [checkIfMethodIsEnabled](https://github.com/MetaMask/metamask-extension/blob/develop/ui/selectors/selectors.js#L328) that we can use here.